ci-operator/step-registry/azure/provision/service-principal/hypershift/azure-provision-service-principal-hypershift-commands.sh

```diff
@@ -40,6 +46,12 @@ for component in $COMPONENTS; do
     component_to_cert_name+=(["$component"]="$name")
 done
 
+# TODO: Remove this once the we used the automated role assignment by "--assign-service-principal-role"
+az role assignment create \
+  --assignee "${component_to_client_id[ingress]}"\
+  --role "Contributor" \
+  --scope  /subscriptions/"$AZURE_AUTH_SUBSCRIPTION_ID"/resourceGroups/os4-common
```

Comment:
`os4-common` looks like a hard-code, you can introduce an ENV var - `BASE_DOMAIN_RESOURCE_GROUP` with default value - "os4-common" to avoid it.

Reply:
Fixed.